enterprise/admin/views.py

```diff
@@ -199,8 +200,19 @@ def _enroll_users(cls, emails, course_details, mode, request):
             ).format(enrolled_count=enrolled_count, course_id=course_id))
         if non_existing:
             messages.warning(request, _(
-                "The following users do not have accounts yet and were not enrolled: {}"
+                "The following users do not have accounts yet and were not yet enrolled: {}"
```

Comment:
@srpearce, this message is used when an action is taken which would result in enrollment. In other words, the second clause isn't a conditional - it's the result of the operation.

Similar to the phrase: "The following users are lactose-intolerant, and were not given ice cream cones:"